src/external/modules/view-licences/controller.js

```diff
@@ -158,7 +158,7 @@ const getCommonLicenceViewContext = async (licenceNumber, documentId, documentNa
     documentId,
     ...returnsData,
     pageTitle: getPageTitle(documentName, licenceNumber),
-    back: `/admin/licences?query=${licenceNumber}`,
+    back: `/licences?query=${licenceNumber}`,
```

Comment:
Unless it's been changed elsewhere, the `?query=` is not a valid query param for the external licences view and causes a 500 error.  For externals there are 2 query params `emailAddress` and `licenceNumber`